### ISSUE TITLE
Chunked phase-retrieval reconstruction incorrectly recover slices on borders

### ISSUE THREAD

**Issue by nikitinvv** [tomography/tomopy-cli#38]
Example:
If chunk size 32 then slices 0..4 and 28..31 shows no effect of the Paganin filtering.

For correct filtering one should use the chunks of sizes at least 48 (40 for ichunk=0 and ichunk=end-1), i.e. 0..40, 24..72, and cut them after filtering to get 0..32, 32..64, etc.

**Comment by nikitinvv** [comment]
Solved in this [commit](https://github.com/tomography/tomopy-cli/commit/7fb4d6904ae48e1416c00a2595abc206d997f7ba)

**Comment by nikitinvv** [comment]
after/before: ![Screenshot at 2020-03-25 01-52-00](https://user-images.githubusercontent.com/3603885/77490829-59171700-6e3c-11ea-82d0-e67379c89880.png)

**Comment by decarlof** [comment]
@nikitinvv shall I add this in the readme file or start a sphynx/readthedoc? Perhaps the last is better so we can also add better documentation for other functionalities (see hardening added by @aps-7bm etc.)

**Comment by nikitinvv** [comment]
@decarlof, agree. it would be good to write about that in readthedocs. I think this problem may also occur in beam hardening
